core/services/relay/evm/evm.go

```diff
@@ -71,6 +72,7 @@ var _ commontypes.Relayer = &Relayer{} //nolint:staticcheck
 
 type Relayer struct {
 	db          *sqlx.DB
+	ds          sqlutil.DataSource
```

Comment:
can you add a comment here about why we need both the db and the ds? one the surface it seems redundant.